.drone.star

```diff
@@ -14,10 +14,10 @@ def main(ctx):
     },
 
     {
-      'value': '10.4.0-rc3',
-      'qa': 'https://download.owncloud.org/community/testing/owncloud-10.4.0RC3-qa.tar.bz2',
-      'tarball': 'https://download.owncloud.org/community/testing/owncloud-10.4.0RC3.tar.bz2',
-      'tarball_sha': '4dbff6d09aec22d6d3a13f86c86c6b79a17ffd32a7140e94bd06b1f8ed43206b',
+      'value': '10.4.0',
```

Comment:
please also make these changes to `latest`

Reply:
ok. 
I should probably also add a "tag" 10.4 to the 10.4.0 build. 
Should we move the "tag" 10 from 10.3.2 to 10.4.0 too? 

It is good to remove 10.2.1 from this build.
Is there an explanation, why we did rebuild 10.3.2 and 10.2.1 all the time?
I'd like to stop rebuilding 10.3.2 too, because in my understanding a release is a release and should not be re-released. Any small change that may sneak in, could frustrate reproducers.

Reply:
good point, we release the base image from time to time.
Some weeks ago, this was necessary. 
It would not change the oc10 Server Code